experiments/resources/back_up_optimizely.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -exuo pipefail
+
+# Required by click http://click.pocoo.org/5/python3/
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+export GIT_AUTHOR_NAME = $GIT_AUTHOR_NAME
+export GIT_AUTHOR_EMAIL = $GIT_AUTHOR_EMAIL
+
+cd $WORKSPACE/py-opt-cli
+pip install -r requirements.txt
+pip install -e .
+env
+
+cd $WORKSPACE/optimizely-experiments
+py-opt-cli pull
+
+git add -A
+git commit -am "history commit!"
+git push origin HEAD:history
```

Comment:
Looks like your editor didn't add a newline to the end of the file. While not mandatory it's good style to have the file end with a new line and if you don't git/github complain about it.